saleor/graphql/product/tests/mutations/test_product_update.py

```diff
@@ -278,6 +278,44 @@ def test_update_product_doesnt_clear_description_plaintext_when_no_description(
     assert product.description_plaintext == description_plaintext
 
 
+def test_update_product_seo_fields(
```

Comment:
We should add the same test for `seo_description` field.

Reply:
added